docs/upgrading-client.md

```diff
@@ -0,0 +1,52 @@
+<!--[metadata]>
++++
+title = "Upgrading notary client"
```

Comment:
nit: It seems that other titles use capitalized `Notary`, so we might want to update this usage here too for consistency